modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -83,6 +83,84 @@ definition {
 		}
 	}
 
+	@description = "LPS-149051 - Verify that it's possible to delete a notification"
+	@priority = "4"
+	test CanDeleteNotification {
+		task ("Given a template is created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				subject = "Subject");
+
+			Button.clickSave();
+		}
+
+		task ("Given an Object and an attachment field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161922",
+				objectName = "CustomObject161922",
+				pluralLabelName = "Custom Objects 161922");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Attachment",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "Long",
+				fileSource = "userComputer",
+				isRequired = "false",
+				objectName = "CustomObject161922",
+				showFilesInDocumentsAndMedia = "true");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject161922");
+		}
+
+		task ("Given an action is created") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 161922");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Custom Action",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "Action shot on add",
+				notificationName = "Notification Template",
+				objectLabel = "Custom Object 161922",
+				thenAction = "Notification",
+				whenAction = "On After Add");
+		}
+
+		task ("When an entry is created") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject161922",
+				value = "Entry A");
+		}
+
+		task ("Then delete the notification and assert the notification is no longer showed") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Queue");
+
+			ObjectNotifications.deleteQueue(triggerBy = "CustomObject161922");
+
+			AssertElementNotPresent(
+				key_text = "CustomObject161922",
+				locator1 = "Icon#ANY_VERTICAL_ELLIPSIS");
+		}
```

Comment:
`Then the notification is no longer present on the page`

@evanilson-santana  do we have a better locator to use? Ideally, we don't want to use the ellipsis here, it would be better to use something like the name of the notification on the table.